plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/discount/index.tsx

```diff
@@ -49,7 +49,15 @@ const TotalsDiscount = ( {
 	} = values;
 	const discountValue = parseInt( totalDiscount, 10 );
 
-	if ( ! discountValue && cartCoupons.length === 0 ) {
+	// We need to run the filter even if we don't have coupons.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

Comment:
Is this necessary? The var is used straight away

Reply:
Ah yeah good point, it was not used before.